src/Compilers/Core/Portable/Symbols/Attributes/CommonParameterEarlyWellKnownAttributeData.cs

```diff
@@ -80,6 +80,38 @@ public bool HasCallerMemberNameAttribute
                 SetDataStored();
             }
         }
+
+        private bool _hasCallerArgumentExpressionAttribute;
```

Comment:
>private bool _hasCallerArgumentExpressionAttribute; [](start = 8, length = 51)

Unless we will support this attribute in VB, the new code here should be moved to C# derived type.

Reply:
VB support is still an open question.

Reply:
> VB support is still an open question.

Then I suggest to either moving this code to C# specific type, or adding a prototype comment to follow up on this later, once we make the decision.

---
In reply to: [600238389](https://github.com/dotnet/roslyn/pull/51952#discussion_r600238389) [](ancestors = 600238389)

Reply:
Added a prototype comment.